alf/networks/containers.py

```diff
@@ -51,34 +101,70 @@ def Sequential(*modules, input_tensor_spec=None, name="Sequential"):
     # does not allow *args for __init__() (see _NetworkMeta.__new__()). And we
     # want to use *modules here to make the interface consistent with
     # torch.nn.Sequential and alf.layers.Sequential to avoid confusion.
-    return _Sequential(modules, input_tensor_spec=input_tensor_spec, name=name)
+    return _Sequential(
+        modules,
+        named_modules,
+        output=output,
+        input_tensor_spec=input_tensor_spec,
+        name=name)
+
+
+def _get_nested_field(nest, nest_fields):
```

Comment:
This function also appears in layers.py and can potentially be used in other cases as well.
Maybe we can move it to nest.utils?